### PR TITLE
Update marshalling.md

### PR DESCRIPTION
**Pull request by mb720**
Grammar, wording, typo.

### CODE REVIEW

**Comment by jlprat** [comment]
OK TO TEST

**Comment by jlprat** [comment]
Thanks for the PR!!

File: docs/src/main/paradox/scala/http/common/marshalling.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # Marshalling
 
-"Marshalling" is the process of converting a higher-level (object) structure into some kind of lower-level
-representation, often a "wire format". Other popular names for it are "Serialization" or "Pickling".
+Marshalling is the process of converting a higher-level (object) structure into some kind of lower-level
+representation, often a "wire format". Other popular names for marshalling are "serialization" or "pickling".
```
Comment (jlprat) [inline docs/src/main/paradox/scala/http/common/marshalling.md]: I'm not so sure about not capitalizing the quoted serialization and pickling. But maybe is the German influence that biases me.

Comment (mb720) [inline docs/src/main/paradox/scala/http/common/marshalling.md]: I have to fight German bias too :-)
But I'd argue that according to [the rules of capitalization](https://en.wikipedia.org/wiki/Capitalization_in_English) those words should be lowercase.

Comment (jlprat) [inline docs/src/main/paradox/scala/http/common/marshalling.md]: Yes, I think your version is right.

**Comment by mb720** [comment]
You're welcome!
:-)

**Comment by mb720** [comment]
Whoops. Closed prematurely.

**Comment by jlprat** [comment]
Usually, we need two approvals to merge, but now being summertime, things are little slower. I'll wait a day or two, just in case somebody else wants to review it, then I'll merge.

**Pull request merged**